platform/source/ATCmdParser.cpp

```diff
@@ -337,6 +264,31 @@ bool ATCmdParser::vrecv(const char *response, std::va_list args)
         }
     }
 
+    return 1;
+}
+
+// Command parsing with line handling
+bool ATCmdParser::vsend(const char *command, std::va_list args)
```

Comment:
Why was this moved? The header file has this one declared before vrecvscanf. Perhaps we could keep the header's order?

Reply:
Done